src/components/organisms/SettingsManager/SettingsManager.tsx

```diff
@@ -183,6 +201,26 @@ const SettingsManager: React.FC = () => {
               Show Cluster Selector
             </Checkbox>
           </S.Div>
+          <S.Div>
+            <S.Span style={{display: 'inline-block', marginRight: '8px'}}>Telemetry</S.Span>
+            <Tooltip title="DOCUMENTATION URL">
```

Comment:
should this link to something!?

Reply:
We talked to point to the privacy policy by this tooltip - but it is not ready yet.

Reply:
We are preparing it with Lucky. He has already a template and we will decide how to do it